### DISCUSSION TITLE
Collapse arrow acts weird when padding is removed

### DISCUSSION THREAD

**Discussion by HOMA40K** [saadeghi/daisyui#2914] [Q&A] [answered]
So, I want to make collapse item smaller, so it becomes the same size as other items, but I didn't find the way to change arrows height

![image](https://github.com/saadeghi/daisyui/assets/55148431/2fa96473-a918-4d01-8edc-929f7c89694a)

``` html
<div tabindex="0" class="collapse collapse-arrow border border-base-300 bg-base-200 peer-checked:text-secondary-content">
         <input type="checkbox" class="peer min-h-0" />
         <div class="collapse-title text-xl font-medium p-0 min-h-0">
                Focus me to see content
         </div>
         <div class="collapse-content peer-checked:text-secondary-content">
                <p>tabindex="0" attribute is necessary to make the div focusable</p>
         </div>
</div>
```

**Comment by Vijaykv5** [comment]
Hi @HOMA40K 
To adjust the collapse arrow's size in Tailwind CSS while maintaining consistency with other items, ensure the collapse item's padding is consistent across all elements. Also Make sure to customize the arrow's size directly through Tailwind's `h-[size]` and `w-[size]` utilities, ensuring it aligns properly within the collapse item.

**Reply by HOMA40K** [reply]
![image](https://github.com/saadeghi/daisyui/assets/55148431/14bb6b5d-2d7f-4bb6-b04b-e51472005f23)
it seems that this arrow is completely different thing, because after adding padding it doesn't move even a bit

```html
<div tabindex="0"
            class="collapse collapse-arrow border border-base-300 bg-base-200 peer-checked:text-secondary-content">
            <input type="checkbox" class="peer min-h-0" />
            <div class="collapse-title text-xl font-medium p-0 min-h-0">
                Focus me to see content
            </div>
            <div class="collapse-content peer-checked:text-secondary-content ">
                <p>tabindex="0" attribute is necessary to make the div focusable</p>
            </div>
        </div>
```

**Reply by Vijaykv5** [reply]
Does this fixes it? I hope you need the arrow to fixed after removing padding?
May you can be more specific on your issue? @HOMA40K

**Reply by HOMA40K** [reply]
No, I need to move this this arrow

**Comment by saadeghi** [comment, marked as answer]
I added `after:!top-4` class to your code:  
https://play.tailwindcss.com/Nhfn77BFVS
```html
<div tabindex="0" class="collapse collapse-arrow border border-base-300 bg-base-200 peer-checked:text-secondary-content">
  <input type="checkbox" class="peer min-h-0" />
  <div class="collapse-title min-h-0 p-0 text-xl font-medium after:!top-4">Focus me to see content</div>
  <div class="collapse-content peer-checked:text-secondary-content">
    <p>tabindex="0" attribute is necessary to make the div focusable</p>
  </div>
</div>
```

**Reply by HOMA40K** [reply]
Huge thanks to you
